src/itest/java/uk/gov/companieshouse/company/links/steps/CompanyLinksSteps.java

```diff
@@ -0,0 +1,151 @@
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.io.FileUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.util.ResourceUtils;
+import uk.gov.companieshouse.company.links.service.CompanyProfileService;
+import uk.gov.companieshouse.stream.EventRecord;
+import uk.gov.companieshouse.stream.ResourceChangedData;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CompanyLinksSteps {
+
+    @Value("${wiremock.server.port}")
+    private String port;
+
+    @Value("${company-links.consumer.insolvency.topic}")
+    private String topic;
+
+    private static WireMockServer wireMockServer;
+
+    private String companyNumber;
+
+    @Autowired
+    private CompanyProfileService companyProfileService;
+
+    @Autowired
+    public KafkaTemplate<String, ResourceChangedData> kafkaTemplate;
+
+    @Given("Company links consumer api service is running")
+    public void company_links_consumer_api_service_is_running() {
+        assertThat(companyProfileService).isNotNull();
+    }
+
+    @When("a message is published for companyNumber {string} to update links")
+    public void a_message_is_published_for_company_number_to_update_links(String companyNumber) throws InterruptedException {
+        this.companyNumber = companyNumber;
+        configureWiremock();
+
+        stubUpdateConsumerLinks();
+        kafkaTemplate.send(topic, createMessage(this.companyNumber));
+
+        CountDownLatch countDownLatch = new CountDownLatch(1);
+        countDownLatch.await(5, TimeUnit.SECONDS);
+    }
+
+    @When("a message is published for companyNumber {string} to check for links")
+    public void a_message_is_published_for_company_number_to_check_for_links(String companyNumber) throws InterruptedException {
+        this.companyNumber = companyNumber;
+        configureWiremock();
+
+        stubGetConsumerLinks();
+        kafkaTemplate.send(topic, createMessage(companyNumber));
+
+        CountDownLatch countDownLatch = new CountDownLatch(1);
+        countDownLatch.await(5, TimeUnit.SECONDS);
+    }
+
+    @Then("the Company Links Consumer should send a GET request to the Company Profile API")
+    public void the_company_links_consumer_should_send_a_get_request_to_the_company_profile_api() {
+        verify(1, getRequestedFor(urlPathMatching("/company/([0-9]*)")));
+        verify(0, patchRequestedFor(urlPathMatching("/company/([0-9]*)/links")));
+
+        wireMockServer.stop();
+    }
+
+    @Then("the Company Links Consumer should send a PATCH request to the Company Profile API")
+    public void the_company_links_consumer_should_send_a_patch_request_to_the_company_profile_api() {
+        verify(1, getRequestedFor(urlPathMatching("/company/([0-9]*)")));
```

Comment:
sorry, CH changed GET url as well to company/{company_number}/links (same as PATCH)